packages/webpack/README.md

```diff
@@ -81,6 +81,12 @@ Every constant's value is `JSON.stringify()`-ed first, so you don't have to reme
 
 Special feature: Using `defineConstants` multiple times results in a single DefinePlugin instance configured to do all the replacements.
 
+#### setEnv(constants: string[]|object): Function
+
+Replaces constants in your source code with a values from `process.env` using the [webpack.EnvironmentPlugin](https://webpack.js.org/plugins/environment-plugin/).
+
+Special feature: Using `setEnv` multiple times results in a single EnvironmentPlugin instance configured to do all the replacements.
```

Comment:
- “Special feature” → “Note”.
- Couple of examples would be nice to have here.

Reply:
Copied directly from the `defineConstants` block above.

Reply:
Then we’ll need to change it there too, there’s nothing *special* that, just a little convenience ;-)